cognitedata/reveal
# fix: point cloud performance regression for certain platforms when doing model intersection

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-819

## Description :pencil:

Performance regressions were observed in Chrome (Android, MacOS, iOS) and Safari (MacOS, iOS) when `getIntersectionFromPixel` method was called, reducing Reveal fps every time it is called (see Jira issue for video). This PR resolves this problem by not using `cancelAnimationFrame` method but rather a simple flag to temporarily stop rendering.

## How has this been tested? :mag:

Tested in viewer examples on Chrome (MacOS, Android) with different point cloud models.



## Test instructions :information_source:

1. Load any point cloud model in viewer examples on android/iOS or MacOS
2. Click 10 or more times on the model and observe no change in fps.



## Checklist :ballot_box_with_check:
